docs/docs/api/http.md

```diff
@@ -643,3 +643,22 @@ The response comes in two parts:
   - `total`
 
     The total number of elements across all pages.
+
+### Metadata
+
+Airy provides a mechanism to add metadata to a conversation. A metadata entry consists of a key and a value.
+A key can use the dot notation to represent namespaces.
```

Comment:
I would have the defintions in the glossary and then refer to them from here (sorry I didn't realise I could have suggested this in the previous pass).